README.rst

```diff
@@ -46,7 +46,11 @@ something like the following code somewhere in your app files:
     process. Even though there are multiple modules loaded automatically,
     only the ``models`` modules (usually the ``models.py`` file of your
     app) are guaranteed to be loaded at startup. Therefore it's recommended
-    to put your ``AppConf`` subclass(es) there, too.
+    to put your ``AppConf`` subclass(es) there, too; or that you import these 
+    classes in your Django AppConfig ready_ method.
```

Comment:
I think I'd drop this change. I'm not sure it is right in general. `ready()` is not called until after the app registry is loaded. Often we want settings prior to that... _It depends_.